src/log/Entry.h

```diff
@@ -41,7 +41,6 @@ class Entry {
   pthread_t m_thread;
   short m_prio, m_subsys;
 
-private:
   static log_clock& clock() {
```

Comment:
This is called in the `Entry` constructor. Why undo `private:`?

Reply:
Now, it’s called in Log::set_coarse_timestamp.